docs/Deployment.md

```diff
@@ -0,0 +1,94 @@
+|--------------------------|----------|------------------------|-------------------------------------------------------------------------------------------------------|---------------------|
+| Max Partition Count      | uint16_t | MaxPartitionCount      | The maximum processor count used for partitioning work in MsQuic                                      | Yes                 |
+| Max Operations per Drain | uint8_t  | MaxOperationsPerDrain  | The maximum number of operations to drain per connection quantum                                      | No                  |
+| Retry Memory Limit       | uint16_t | RetryMemoryFraction    | The percentage of available memory usable for handshake<br>connections before stateless retry is used | No                  |
+| Max Worker Queue Delay   | uint32_t | MaxWorkerQueueDelayMs  | The maximum queue delay (in ms) allowed for a worker thread                                           | No                  |
+| Max Stateless Operations | uint32_t | MaxStatelessOperations | The maximum number of stateless operations that may be queued at<br>any one time                      | No                  |
+| Initial Window Size      | uint32_t | InitialWindowPackets   | The size (in packets) of the initial congestion window for a<br>connection                            | No                  |
+|                          |          |                        |                                                                                                       |                     |
+
+**TODO** - Finish list above
+
+## Windows
+
+On Windows, these settings can set via the registry. The main registry path for the keys is:
+
+> HKLM:\System\CurrentControlSet\Services\MsQuic\Parameters
+
+The settings can also be set per "app-name" (as indicated in `RegistrationOpen`):
+
+> HKLM:\System\CurrentControlSet\Services\MsQuic\Parameters\app-name
+
+The `DWord` type should be used for all 32-bit or less types. For 64-bit types, `DWord` or `QWord` may be used. If invalid types or values are used, they will be ignored and the built-in default will be used instead.
+
+# Cipher Suites
+
+## Windows
+
+By default, the new cipher suite `TLS_CHACHA20_POLY1305_SHA256` is disabled. It can be enabled via the following command:
+
+```PowerShell
+Enable-TlsCipherSuite -Name TLS_CHACHA20_POLY1305_SHA256
+```
+
+# Firewall
+
+## Windows
+
+In order to configure the Windows firewall to allow inbound QUIC traffic efficiently, use a command such as the one below.
+
+```PowerShell
+New-NetFirewallRule -DisplayName "Allow QUIC" -Direction Inbound -Protocol UDP -LocalPort 433 -Action Allow -LocalOnlyMapping $true
+```
+
+Note the use of the `-LocalOnlyMapping $true` argument. This is a performance optimizing feature that should be used for UDP based protocols (like QUIC).
```

Comment:
Might be good to explain what this does rather then just saying its a performance optimization. Just saying set this flag for better perf isn't usually helpful.

Reply:
I added a link to MSDN. They don't really same much either. I can't actually tell you what this does myself...

Reply:
Ok. Well at least now it points to where info would be.